charts/code-server/README.md

```diff
@@ -0,0 +1,118 @@
+# code-server
```

Comment:
Can you add a note that this is community maintained? Neither myself nor @code-asher are familiar with helm.

If yourself @Matthew-Beckett or @alexgorbatchev would like become the official maintainers of the helm chart, I would be happy with that as well and you can add a note here.